main.py

```diff
@@ -539,8 +548,8 @@ def main():
 
               # we only take the last-saved accs and losses from train
               if epoch_step % 50 == 0:
-                  accuracy_train = (outputs.detach().argmax(-1) == targets).float().mean().item()
-                  loss_train = loss.cpu().item()/batchsize
+                  train_acc = (outputs.detach().argmax(-1) == targets).float().mean().item()
+                  train_loss = loss.detach().cpu().item()/batchsize
```

Comment:
Not sure how I missed this before, but this could have caused a few issues, especially if I ended up 'cleverly' doing inplace computational stuff on the loss before printout at some point in the future, or did something like appending it to a list that held the graph around even longer.